### ISSUE TITLE
overdiagnosis

### ISSUE THREAD

**Issue by max-hffmstr** [obophenotype/human-phenotype-ontology#11340]
**Preferred term label: Overdiagnosis**

**Excessive use of diagnostics, overuse of diagnostic methods and screening, sometimes induced by patients demands PMID: 29367314**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (psychogenic movement disorders: ORPHA:71519)**

**Your nano-attribution (ORCID: 0009-0009-5950-9366)**

**Comment by pnrobinson** [comment]
@max-hffmstr I agree this is a major issue in modern healthcare, but it is not an observable phenotypic feature (basically, it is the inference that a diagnosis made by a physician will not positively impact on a person's health for one of many reasons). This is outside the scope of the HPO.
